cpp/include/cugraph/detail/utility_wrappers.hpp

```diff
@@ -73,11 +73,10 @@ void scalar_fill(raft::handle_t const& handle, value_t* d_value, size_t size, va
  * @param [in]  handle RAFT handle object to encapsulate resources (e.g. CUDA stream, communicator,
  * and handles to various CUDA libraries) to run graph algorithms.
  * @param[out]  d_value      device array to sort
```

Comment:
Update documentation to reflect this is a span, maybe rename d_span?  Same for all 3 functions.

Reply:
Done